scripts/adder-collator.sh

```diff
@@ -31,11 +31,11 @@ adder_collator="target/release/adder-collator"
 
 # ensure the polkadot binary exists and is up to date
 if [ ! -x "$polkadot" ] || [ "$polkadot" -ot "$last_modified_rust_file" ]; then
-  cargo build --release --features real-overseer
+  cargo +nightly build --release --features real-overseer
```

Comment:
what required nightly?

Reply:
https://github.com/rust-lang/rust/issues/75992 causes exponential compile time, so compilation for testing took _ages_.

Reply:
but this is fixed in 1.48 which is latest stable, right?

Reply:
I just fixed my local toolchain which was stuck to 1.46 for some reason, so I did not test against 1.48